lib/features/manage_account/presentation/profiles/identities/utils/identity_utils.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:jmap_dart_client/jmap/core/unsigned_int.dart';
+import 'package:jmap_dart_client/jmap/identities/identity.dart';
+
+class IdentityUtils {
+  
+  static Identity? getSmallestOrderedIdentity(List<Identity>? identities) {
+    if (identities == null || identities.isEmpty) {
+      return null;
+    }
+    return identities.fold<Identity>(Identity(sortOrder: UnsignedInt(9007199254740991)), (previousValue, element) {
+      if (previousValue.sortOrder == null || element.sortOrder == null) {
```

Comment:
- one of them is null -> return the remaining
- two of them is null -> return one of them

How about it?